lib/block_producer.go

```diff
@@ -225,9 +225,7 @@ func (desoBlockProducer *DeSoBlockProducer) _getBlockTemplate(publicKey []byte)
 			if err != nil {
 				return nil, nil, nil, errors.Wrapf(err, "Error copying UtxoView: ")
```

Comment:
Go through and cut all long lines down.

Reply:
Won't this error if you pass 0 as the timestamp instead of what you expect to actually put in the block as the timestamp?

Reply:
After reading the _connect functions, I think you definitely need to set the actual block timestamp here FYI.